webpack/scenes/ContentViews/Details/Repositories/ContentCounts.js

```diff
@@ -0,0 +1,60 @@
+import React, { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+
+// type: [singular_name, plural_name, link]
+const repoLabels = {
+  rpm: ['rpm packages', 'rpm package', 'packages'],
+  module_stream: ['module streams', 'module stream', 'module_streams'],
+  erratum: ['errata', 'erratum', 'errata'], // need to handle link, its $URL/errata?repositoryId=107
+  deb: ['deb packages', 'deb package', 'debs'],
+  ansible_collection: ['ansible collections', 'ansible collection', 'ansible_collections'],
+  docker_manifest: ['container manifests', 'container manifest', 'content/docker_manifests'],
+  docker_manifest_list: ['container manifest lists', 'container manifest list', 'content/docker_manifest_lists'],
+  docker_tag: ['container tags', 'container tag', 'content/docker_tags'],
+  file: ['files', 'file', 'content/files'],
+  ostree_branch: ['ostree branches', 'ostree branch', 'content/ostree_branches'],
+  package_group: ['package groups', 'package group', 'package_groups'],
+  puppet_module: ['puppet modules', 'puppet module', 'content/puppet_modules'],
+  srpm: ['source rpms', 'source rpm', 'source_rpms'], // no link?
+};
+
+const appendCount = (type, count, info, productId, repoId) => {
+  const [repoPlural, repoSingular, link] = info;
+  const displayName = count > 1 ? repoPlural : repoSingular;
+  let url = urlBuilder(`products/${productId}/repositories/${repoId}/content`, '', link);
+  const displayInfo = <Fragment>{`${count} ${displayName}`}</Fragment>;
+  if (type === 'source_rpm') return displayInfo;
+  if (type === 'erratum') url = urlBuilder(`errata?repositoryId=${repoId}`);
+
+  return {
+    key: `${type}${count}`,
+    component: <Link to={url}>{displayInfo}</Link>,
```

Comment:
The `packages` link isn't working either.  (see the other comment about the links)